zkvm/src/txlog.rs

```diff
@@ -101,7 +102,7 @@ impl Entry {
                 t.commit_u64(b"nonce.t", *maxtime);
             }
             Entry::Output(outstruct) => {
-                t.commit_bytes(b"output", outstruct);
+                t.commit_bytes(b"output", &outstruct.clone().to_bytes());
```

Comment:
Per comment above, if `Output::to_bytes()` takes &self, we don't need to clone the output here.